### ISSUE TITLE
Do not use inspect.getargspec() in twisted/names/dns.py on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8703]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8703|
|Type|enhancement|
|Created|2016-07-28 21:37:36Z|
|Branch|https://github.com/twisted/twisted/tree/8703-rodrigc-getargspec-py3|

inspect.getargspec() is deprecated on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8703 8703
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8703_rodrigc_getargspec_py3 8703-rodrigc-getargspec-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469741856354733 1469741856354733
changetime__1470694695811508 1470694695811508
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/443

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

I have a review note, I'm afraid.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

Review updated: https://github.com/twisted/twisted/pull/443#discussion_r73843147

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_5"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Ok, I've re-reviewed. You let a few lines of extra whitespace slip in, which I'd like trimmed, but otherwise this seems good: fix the whitespace up and merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub set status to `closed`|
|-|-|

In changeset 88d14580e6ff4da4be8d84440852b365d427aa31
```
#!CommitTicketReference repository="" revision="88d14580e6ff4da4be8d84440852b365d427aa31"
Merge pull request #443 from twisted/8703-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8703

Do not use inspect.getargspec() in twisted/names/dns.py on Python 3
```
